### ISSUE TITLE
Configure CI

### ISSUE THREAD

**Issue by Lotes** [Lotes/CQL#5]
- [ ] tests
- [ ] test coverage
- [ ] documentation

**Comment by igitur** [comment]
I have set up a branch for a basic CI process on AppVeyor, but if you accept #7 , then I will need to do some work on it. I will log a new PR for that after #7 is resolved.
